libraries/Microsoft.Bot.Builder.Dialogs/DialogContextExtension.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Globalization;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    /// <summary>
+    /// Provides extension methods for <see cref="DialogContext"/>.
+    /// </summary>
+    public static class DialogContextExtension
```

Comment:
Maybe this class could be merged into the "Extensions" class ?

Reply:
merged into DialogExtensions